### ISSUE TITLE
Support: Add suport for Mozilla Firefox

### ISSUE THREAD

**Issue by gupta-shrinath** [dheerajdlalwani/back-to-work#10]
I was trying to install this extension on firefox(primary browser) but it seems that Manifest v3 is not supported.
![image](https://user-images.githubusercontent.com/42563130/151301924-b9b7dea3-78ee-422c-ad79-b7aca00345a3.png)
Mozilla on Manifest v3 support
https://blog.mozilla.org/addons/2021/05/27/manifest-v3-update/
https://blog.mozilla.org/addons/2019/09/03/mozillas-manifest-v3-faq/
Possible solution:
Downgrade version v2 to support firefox. I have not read the source code of the extension so I don't know if that would be a problem.
I'll be happy to work on this if any major work is required.

**Comment by dheerajdlalwani** [comment]
Okay, let me think this over. If you have any suggestions on how to approach this, feel free to comment down..
Thank you so much!

**Comment by gupta-shrinath** [comment]
First, we need to know whether the extension(code) runs in Manifest v2 and if so then it would run on firefox as well IMO. If you check this in your local will have a clear idea. I'm not sure whether this is the best solution but it is a simple solution.

**Comment by dheerajdlalwani** [comment]
No, this particular version uses certain APIs which are specific to Manifest V3.
[DeclarativeNetRequest API](https://developer.chrome.com/docs/extensions/reference/declarativeNetRequest) to be one of the major ones. Now Chrome actually has given support for this API in Manifest V2 also, so that it aids the developers in transition from Manifest V2 to V3. I don't know if this is the case with Firefox.
